lib/devtools/flay.rb

```diff
@@ -0,0 +1,92 @@
+module Devtools
+  module Flay
+    # Measure flay mass relative to size of duplicated sexps
+    class Scale
+      include Adamantium
+      include Anima.new(:minimum_mass, :files)
+      include Procto.call(:measure)
+
+      # Measure duplication mass
+      #
+      # @return [Array<Rational>]
+      #
+      # @api private
+      def measure
+        flay.masses.map do |hash, mass|
+          Rational(mass, flay.hashes.fetch(hash).size)
+        end
+      end
+
+      # Report flay output
+      #
+      # @return [undefined]
```

Comment:
Cool I like this. Soon we need to generalize all tools in kind of a `Devtools::Result` object. This will allow to unify the various tools interfaces for parallelized processing.